sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1263,15 +1277,28 @@ public <T> Mono<CosmosItemResponse<T>> readItem(String itemId, PartitionKey part
     }
 
     /**
-     * Reads an item using a configured {@link CosmosItemRequestOptions}.
+     * Reads an item by itemId using a configured {@link CosmosItemRequestOptions}.
```

Comment:
```suggestion
     * Reads an item by Id and partitionKey using a configured {@link CosmosItemRequestOptions}.
```

Reply:
Thanks @kushagraThapar for your feedback, I've removed unnecessary code examples from docs and added them to the samples class. 
Pushed new changes. Please tell me what else needs to be fixed